src/components/GoogleMap.vue

```diff
@@ -154,55 +155,78 @@ export default defineComponent({
       }
     });
 
-    // Only run this in a browser env since it needs to use the `document` object
-    // and would error out in a node env (i.e. vitepress/vuepress SSR)
-    if (typeof window !== 'undefined') {
-      const loader = new Loader({
-        apiKey: props.apiKey,
-        version: 'weekly',
-        libraries: props.libraries || ['places'],
-        language: props.language,
-        region: props.region,
+    const onLoaded = () => {
+      // eslint-disable-next-line no-undef
+      const { Map } = (api.value = google.maps);
+      console.log('MAPREF:::', mapRef, google)
```

Comment:
is this meant to be here? Looks like debugging code.

Reply:
removed